include/openthread/thread_ftd.h

```diff
@@ -355,6 +355,29 @@ uint8_t otThreadGetRouterUpgradeThreshold(otInstance *aInstance);
  */
 void otThreadSetRouterUpgradeThreshold(otInstance *aInstance, uint8_t aThreshold);
 
+/**
+ * Get the MLE_CHILD_ROUTER_LINKS parameter used in the REED role.
+ *
+ * @param[in]  aInstance A pointer to an OpenThread instance.
+ *
+ * @returns The MLE_CHILD_ROUTER_LINKS value.
+ *
+ * @sa otThreadSetChildRouterLinks
+ *
+ */
+uint8_t otThreadGetChildRouterLinks(otInstance *aInstance);
+
+/**
+ * Set the MLE_CHILD_ROUTER_LINKS parameter used in the REED role.
+ *
+ * @param[in]  aInstance         A pointer to an OpenThread instance.
+ * @param[in]  aChildRouterLinks The MLE_CHILD_ROUTER_LINKS value.
+ *
+ * @sa otThreadGetChildRouterLinks
+ *
+ */
+void otThreadSetChildRouterLinks(otInstance *aInstance, uint8_t aChildRouterLinks);
```

Comment:
Suggest returning `otError` from this API:
```
 * @retval OT_ERROR_NONE            Successfully set the value.
 * @retval OT_ERROR_INVALID_STATE   Thread protocols are enabled.
 *

Reply:
done.